sdcm/cluster.py

```diff
@@ -89,6 +89,8 @@
 INSTANCE_PROVISION_ON_DEMAND = 'on_demand'
 SPOT_TERMINATION_CHECK_DELAY = 5
 
+BUILD_ID_RE = re.compile(r"Build ID: (?P<build_id>\w+)")
```

Comment:
better to move this to `sdcm/utils/version_utils.py`

Reply:
done